src/getHumanAge.test.js

```diff
@@ -4,7 +4,59 @@ describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  it('should return an array', () => {
+    expect(getHumanAge(0, 0)).toBeInstanceOf(Array);
+  });
+
+  it(`should return [0, 0] when input 'catAge' is '0' and 'dogAge' is '0'`,
```

Comment:
It would be better to write 
should return [0, 0] when catAge = 0, dogAge = 0

Use it for your following tests